### DISCUSSION TITLE
(emit() problem) emit is not defined

### DISCUSSION THREAD

**Discussion by mittb1g** [livewire/livewire#4030] [1. Help]
1) from **wire:click**
I tried to use the emit function from a wire:click and it doesn't work, and in the console I get this error.

![image](https://user-images.githubusercontent.com/67843213/137617991-26fe4b53-e0c8-4f63-93e8-e5a1428d914e.png)
**CODE:**
![image](https://user-images.githubusercontent.com/67843213/137618192-d55b23b7-4ca1-4766-a667-a898ffa72498.png)

2) from **component**

i'm also unable to send an emit () from my components, as they are not sent (in this case it doesn't give me any error, it just doesn't work)

**CODE:**

**Listener**
![image](https://user-images.githubusercontent.com/67843213/137618032-32fc2e89-ae8c-481c-a8de-178d69084017.png)

**Emit**
![image](https://user-images.githubusercontent.com/67843213/137618161-faec202b-4ae2-4f37-94cc-bef9bb32e8a0.png)

**Comment by joshhanley** [comment]
@mittb1g you are using `emit` instead of `$emit` in your `wire:click`. See the docs here for details https://laravel-livewire.com/docs/2.x/events#from-template

Hope this helps!
